README.md

```diff
@@ -170,21 +175,62 @@ execute `toolbox` to start the server:
 > Toolbox enables dynamic reloading by default. To disable, use the
 > `--disable-reload` flag.
 
-#### Homebrew Users
+</details>
+
+<details>
+
+<summary>Container image</summary>
 
-If you installed Toolbox using Homebrew, the `toolbox` binary is available in your system path. You can start the server with the same command:
+To run the server after pulling the [container image](#installing-the-server):
+
+```sh
+export VERSION=0.11.0 # Use the version you pulled
+docker run -p 5000:5000 \
+-v $(pwd)/tools.yaml:/app/tools.yaml \
+us-central1-docker.pkg.dev/database-toolbox/toolbox/toolbox:$VERSION \
+--tools-file "/app/tools.yaml"
+```
+
+> [!NOTE]
+> The `-v` flag mounts your local `tools.yaml` into the container, and `-p` maps the container's port `5000` to your host's port `5000`.
+
+</details>
+
+<details>
+
+<summary>Source</summary>
+
+To run the server directly from source, navigate to the project root directory and run:
+
+```sh
+go run .
+```
+
+> [!NOTE]
```

Comment:
It doesn't look like any of the notes work when using them in the drop downs. Can we plan text instead of the quote notation and maybe just bold "Note"?

Reply:
Sure, I'll check and update this